### DISCUSSION TITLE
[Docs] Add "Related" section to References

### DISCUSSION THREAD

**Discussion by adevade** [statamic/cms#5149] [General]
It would be nice if below the info of the Modifier, Tag and Variable, there was a "Related" section with similar functions.

For example, if I'm viewing [the `days_ago` modifier](https://statamic.dev/modifiers/days_ago), between the content and the "Docs Feedback" widget there could be a "Related modifiers" listing `hours_ago`, `minutes_ago`, `months_ago`, etc...

I can guess and find it in the sidebar, but it's not always as similar naming as in the example above.

**Comment by jackmcdade** [comment]
Good Idea! I see you opened the issue on the docs repo so anyone following can track this there. https://github.com/statamic/docs/issues/714
